components/ai_chat/core/browser/engine/engine_consumer_oai.cc

```diff
@@ -269,9 +282,94 @@ void EngineConsumerOAIRemote::GenerateAssistantResponse(
   base::Value::List messages =
       BuildMessages(model_options_, truncated_page_content, selected_text,
                     is_video, conversation_history);
+
+  // Perform the main response request
   api_->PerformRequest(model_options_, std::move(messages),
-                       std::move(data_received_callback),
-                       std::move(completed_callback));
+                       data_received_callback, std::move(completed_callback));
```

Comment:
Yeah I don't know that we can use the callback here given we've passed ownership in L293

Reply:
Actually seems like we can as RepeatingCallback is copyable as per [chromium docs](https://source.chromium.org/chromium/chromium/src/+/main:docs/callback.md):
> base::RepeatingCallback<> is created by base::BindRepeating(). This is a callback variant that is copyable that can be run multiple times. It uses internal ref-counting to make copies cheap.